src/DemoUI/src/Views/resources/css/index.css

```diff
@@ -0,0 +1,1865 @@
+    padding: 10px 0 5px;
+    font-size: 14px;
+    color: #5d5d5d;
+}
+
+.pl-filter-method-item {
+    padding: 0 !important;
+    margin-bottom: 10px;
+}
+
+.pl-filter-method-item label {
+    color: #5d5d5d;
+    font-weight: normal;
+}
+
+.pl-methods-tab-wrapper {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 0 83.33333%;
+    flex: 0 0 83.33333%;
+    width: 83.33333%;
+    margin: 0 !important;
+    padding: 0 !important;
+    height: 100%;
+    float: left;
+}
+
+.pl-nav-wrapper {
+    padding: 0 !important;
+    display: flex;
+}
+
+.pl-nav-item {
+    height: 30px;
+    line-height: 30px;
+    padding: 0 !important;
+    margin-right: 30px;
+    font-size: 16px;
+    font-weight: normal;
+    color: #5d5d5d;
+    cursor: pointer;
+    border-bottom: 3px solid transparent;
+}
+
+.pl-nav-item * {
+    pointer-events: none;
+}
+
+.pl-nav-item:hover {
+    color: #2095F2;
+}
+
+.pl-nav-item.selected {
+    color: #2095F2;
+    font-weight: bold;
+    border-bottom: 3px solid #2095F2;
+}
+
+.pl-num-shipping-method-results-wrapper {
+    padding: 0 !important;
+    height: 60px;
+    line-height: 60px;
+    color: #5d5d5d;
+    font-size: 12px;
+}
+
+.pl-shipping-services-message {
+    text-align: center;
+    z-index: 1000 !important;
+    margin-top: 30px;
+    font-size: 1.3rem;
+    color: black;
+    position: relative;
+}
+
+.pl-shipping-services-message div {
+    margin-bottom: 20px;
+}
+
+.pl-shipping-services-message .title {
+    font-size: 3rem;
+}
+
+.pl-shipping-services-message .subtitle {
+    font-size: 2rem;
+}
+
+.pl-shipping-services-message .pl-spinner {
+    position: relative;
+    display: flex;
+}
+
+.hidden {
+    display: none !important;
+}
+
+.pl-table-wrapper {
+    padding: 0 !important;
+    height: calc(100% - 100px);
+    overflow-x: hidden;
+    overflow-y: auto;
+}
+
+.pl-table {
+    width: 100%;
+    border-collapse: separate;
+}
+
+.pl-table-header-wrapper {
+    height: 60px;
+    padding: 0 !important;
+    border-bottom: 1px solid #3595f2;
+    border-right: 2px solid #dcdcdc;
+}
+
+.pl-table-header-wrapper th {
+    height: 60px;
+    padding: 0 5px !important;
+    color: #5d5d5d !important;
+    font-weight: bold !important;
+    background-color: white;
+    position: sticky;
+    position: -webkit-sticky; /* Safari fallback */
+    top: 0;
+    z-index: 400;
+    vertical-align: middle !important;
+    font-size: 12px;
+    word-break: break-word;
+    text-transform: uppercase;
+    text-align: left;
+}
+
+.pl-table-row-wrapper {
+    height: 75px;
+    box-sizing: border-box;
+}
+
+.pl-table-row-wrapper td, .pl-table-header-wrapper th {
+    border-top: 1px solid #dcdcdc !important;
+}
+
+.pl-table-header-wrapper > th:first-child, .pl-table-row-wrapper > td:first-child {
+    border-left: 1px solid #dcdcdc !important;
+}
+
+.pl-table-header-wrapper > th:last-child, .pl-table-row-wrapper > td:last-child {
+    color: #2095F2 !important;
+    border-right: 1px solid #dcdcdc !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)